contrib/cdisetup/venus/venus_unix.go

```diff
@@ -0,0 +1,88 @@
+//go:build !windows
```

Comment:
This shouldn't need build tags as the contrib importer is not included by default and already behind a build tag.

Reply:
Ah, linter runs for all package+arch combinations.